### PR TITLE
Feature: Offset

### PR DESCRIPTION
**Pull request by DennisMatthiasScherf**
Hey there @andrelmlins ,

I've been working on a project lately and needed the offset functionality. So I just implemented it.

### CODE REVIEW

**Comment by DennisMatthiasScherf** [comment]
So, how I understand it, branch 'next' is there to target Vue 3.0, and the 'master' branch it there to target Vue 2.0. Right?

File: src/Column.vue [hunk 1]
```diff
@@ -76,31 +120,47 @@ export default {
...
+.colOffset-xs {
+  margin-left: var(--xsOffset);
+}
+
 @media (min-width: 576px) {
   .colVGR-sm {
     flex-basis: var(--smWidthVGR);
     max-width: var(--smWidthVGR);
   }
+  .colOffset-sm {
```
Comment (andrelmlins) [inline src/Column.vue]: I was interested in leaving all styles with the acronym VGR(vue grid resposive). Would it be possible to make this change?

**Review by andrelmlins — CHANGES_REQUESTED** [review]

**Comment by andrelmlins** [comment]
Very good, thanks for the contribution.

Yes, the next branch is for Vue 3.0.

File: src/Column.vue (hunk 1 above)
Comment (DennisMatthiasScherf) [reply src/Column.vue]: Sure. How about 'colVGR-Offset-sm/xs/.. ?

Comment (andrelmlins) [reply src/Column.vue]: Great!!!

Comment (DennisMatthiasScherf) [reply src/Column.vue]: Done

**Comment by DennisMatthiasScherf** [comment]
@andrelmlins  : Is there a reason which prevents us from merging this into the Vue 2.0 and Vue 3.0 branch. Because - to my knowledge - the implemented functionality does not depend on Vue Versions

**Comment by andrelmlins** [comment]
The Vue 3.0 using `rollup-plugin-vue@6.0.0`. The compile with `rollup-plugin-vue@6.0.0` was not working properly in Vue 2.0. I will go new tests to validate.

**Comment by andrelmlins** [comment]
can i merge this PR?

**Comment by DennisMatthiasScherf** [comment]
I thought you wanted to check the 'rollup-plugin-vue@6.0.0' first. Because we could merge this pr into the branches 'next' and 'master' .

**Comment by DennisMatthiasScherf** [comment]
@andrelmlins Bump

How about the idea to implement the offset functionality into master **and** next branch?
The changes I made are based on a function, which should exist in both versions.

**Comment by andrelmlins** [comment]
Unfortunately we will need to keep the 2 versions, I will merge and generate a version for vue 2.0 and vue 3.0

**Comment by andrelmlins** [comment]
Publish new version

Vue 2: 0.3.0 - latest
Vue 3: 1.1.0 - next

Thanks for the contribution

**Pull request merged**